controllers/argocd/applicationset/constants.go

```diff
@@ -0,0 +1,14 @@
+package applicationset
+
+const (
+	// Values
+	AppSetControllerComponent  = "applicationset-controller"
+	AppSetController           = "argocd-applicationset-controller"
+	AppSetGitlabSCMTlsCert     = "appset-gitlab-scm-tls-cert"
+	AppSetGitlabSCMTlsCertPath = "/app/tls/scm/cert"
+
+	// Commands
+	EntryPointSh     = "entrypoint.sh"
+	LogLevel         = "--loglevel"
+	ArgoCDRepoServer = "--argocd-repo-server"
```

Comment:
@ciiay LogLevel should be in the `common` package and `ArgoCDRepoServer` should be in the repo server package